src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -358,7 +359,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
             eventName: "Composer",
             isEditing: false,
             isReply: !!this.props.replyToEvent,
-            inThread: this.props.relation?.rel_type === RelationType.Thread,
+            inThread: this.props.relation?.rel_type === THREAD_RELATION_TYPE.name,
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?